Source/PeleC.H

```diff
@@ -380,6 +380,15 @@ public:
     amrex::MultiFab& LESTerm,
     amrex::Real flux_factor);
 
+  void updateFluxRegistersLES(
+    bool do_reflux,
+    amrex::Real reflux_factor,
+    const amrex::MultiFab& LESTerm,
+    amrex::Real dt,
+    const amrex::MFIter& mfi,
+    amrex::FabType typ,
+    amrex::FArrayBox flux_ec[AMREX_SPACEDIM]);
```

Comment:
I would change this to an `amrex::Array<>`. Also, is this function needed? Not sure I follow the use case.